<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSettingsDialog</class>
 <widget class="QDialog" name="SerialSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Serial - Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../SerialComponent.qrc">
    <normaloff>:/Serial/logo</normaloff>:/Serial/logo</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="portBox"/>
   </item>
   <item>
    <widget class="QToolButton" name="rescanButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../SerialComponent.qrc">
       <normaloff>:/Serial/rescan</normaloff>:/Serial/rescan</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="baurateBox"/>
   </item>
   <item>
    <widget class="QToolButton" name="connectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../SerialComponent.qrc">
       <normaloff>:/Serial/connect</normaloff>:/Serial/connect</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../SerialComponent.qrc"/>
 </resources>
 <connections/>
</ui>
